mava/systems/jax/config.py

```diff
@@ -189,3 +190,32 @@ def get(self) -> SimpleNamespace:
             )
 
         return SimpleNamespace(**self._built_config)
+
+    def get_local_config(self, component: Type[Component]) -> SimpleNamespace:
+        """Get built config for a single component.
+
+        Args:
+            component: component to provide config for.
+
+        Returns:
+            built config for a single component.
+        """
+        if not self._built:
+            raise Exception(
+                "The config must first be built using .build()"
+                "before calling .get_local_config()."
+            )
+
+        global_config = self._built_config
+        local_config: Dict[str, Any] = {}
+
+        config_class = component.config_class()
+        if not config_class:  # no config class for component
+            return SimpleNamespace()
```

Comment:
Nice quality of life change! Does the `config_class` method in the base component class return `None`? Because then adding configs is no longer a requirement for creating components

Reply:
```
    def config_class() -> Optional[Callable]:
        pass
```
Yes, this is it. But I think writing empty config classes might still be good practice to show future users where to add config is they need it.

Reply:
Sure, but it is convenient for quickly testing stuff